cage_test.go

```diff
@@ -0,0 +1,149 @@
+package evervault_test
+
+import (
+	"bytes"
+	"fmt"
+	"io"
+	"net/http"
+	"os"
+	"testing"
+
+	"github.com/evervault/evervault-go"
+	"github.com/stretchr/testify/assert"
+)
+
+const testCage = "staging-synthetic-cage.app_1bba8ba15402.cages.evervault.dev"
```

Comment:
Is it possible for this to use a mocked server like the other tests (with an attestable cert)?